doc: update Matter known issues (#4970)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

Matter (Project CHIP)
=====================

.. rst-class:: v1-6-0

> KRKNWK-10589: Android CHIPTool crash durring commissioning a Matter device

Reviewer: Typo: durring->during

> KRKNWK-10589: Android CHIPTool crash durring commissioning a Matter device

Reviewer 2: ```suggestion
KRKNWK-10589: Android CHIPTool crashes when commissioning a Matter device
```

>   In some circumstances, Android CHIPTool crashes when trying to connect to the Matter device over Bluetooth LE.

Reviewer 2: ```suggestion
  In random circumstances, Android CHIPTool crashes when trying to connect to a Matter device over Bluetooth LE.
```

  **Workaround** Restart the application and try to commission the Matter device again.
  If the problem persists, clear the application data and try again.

.. rst-class:: v1-6-0

KRKNWK-10387: Matter service is needlessly advertised over Bluetooth LE during DFU
  The Matter samples can be configured to include the support for Device Firmware Upgrade (DFU) over Bluetooth LE.
  When the DFU procedure is started, the Matter Bluetooth LE service is needlessly advertised, revealing the device identifiers such as Vendor and Product IDs.

>   The service should be advertised during the device commissioning only.

Reviewer 2: ```suggestion
  The service is meant to be advertised only during the device commissioning.
```
`Should` might suggest an action from the user.
